Spore ModAPI/Spore/Simulator/cTribeHut.h

```diff
@@ -31,8 +31,8 @@ namespace Simulator
 		/* 1F4h */	ResourceKey mDestructModelMd;
 		/* 200h */	ResourceKey mDestructModelHi;
 		/* 20Ch */	ResourceKey mUndamagedModel;
-		/* 218h */	bool field_218;
-		/* 21Ch */	int field_21C;
+		/* 218h */	bool field_218; // 'true' makes the hut emit harp music
+		/* 21Ch */	int field_21C; // values above 0 make the hut emit a 'zzz' sleeping effect
```

Comment:
You can change the variable names to reflect those comments. For instance `mEmitHarpMusic` and `mIsSleeping`. If you do so, put the comments above the variable, and use three slashes `///` so it appears in the documentation